src/model/User.js

```diff
@@ -1,5 +1,3 @@
-'use strict';
```

Comment:
?

Reply:
Because of eslint error

Reply:
It can't be an ESLint error...

Reply:
![image](https://user-images.githubusercontent.com/43181771/120558768-a14ea380-c408-11eb-9a0b-fefc7d0edd1a.png)